plugin/ochttp/trace.go

```diff
@@ -83,6 +83,7 @@ func (t *traceTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	resp, err := t.base.RoundTrip(req)
 	if err != nil {
 		span.SetStatus(trace.Status{Code: trace.StatusCodeUnknown, Message: err.Error()})
+		span.AddAttributes(trace.BoolAttribute("error", true))
```

Comment:
I'm not sure what the point of this is? Is this required by the specs?

Reply:
This is the way to indicate that this unknown is clearly an error. Attribute is taken from open tracing spec. Any suggestions here? Should there be UNKNOWN_ERROR status and UNKNOWN_UNKNOWN?

Reply:
it's from open tracing specs, not open census =). I've added a mention of it in my specs PR

Reply:
the problem is that `UNKNOWN` is ambiguous. Is it good unknown or bad? When we know for sure - it will be good to indicate it

Reply:
I don't think it is ambiguous. The error message will clarify what went wrong. We have spoken about having an OpenTracing compatibility layer and hopefully we will get that working eventually and then that would be an appropriate place to put this I think. For now, I would prefer not to add it or if you really feel strongly about it, please add it to OpenCensus specs and get approval there and then we can implement it.

Reply:
ok. 

> I would prefer not to add it or if you really feel strongly about it, please add it to OpenCensus specs and get approval there and then we can implement it.

exactly what I'm doing =) this is a POC for test cases proposed there. 

I think I'll pull this `error` thing out for now (maybe later tonight) to separate two discussions

Reply:
error attribute was removed from the spec